### ISSUE TITLE
Cookies sent cross domain

### ISSUE THREAD

**Issue by Simone1234342** [Quitten/Autorize#23]
By default, the cookie that is set is sent with all requests, including requests to entirely different domains.

This seems like somewhat unexpected and unnecessary behavior which can leak cookies if the user is not careful. 

A simple solution might be this:
When using the context menu option "send cookie to Authorize", automatically add "interception filters" option "scope only" for the scope that the cookie was in (only sending the cookie to its original domain seems to be the normal use-case, sending it to other domains should require reconfiguration).

A more extensive solution might be:
Have two fields for the cookie input next to each other. The cookie itself, and the scope of the cookie. This would make it very clear where which cookie is sent. It would also allow to test two scopes in parallel without accidentally disclosing cookies (if more than one entry of cookie/scope pairs is allowed).

**Comment by Quitten** [comment]
Hey, I did it by purpose, I dont want users first time experience will be messy, so by default i add it to any request.
I understand your thoughts and i think that you should set relevant interception filters.
I think your 2nd solution is better, I will implement it once I will fix another issue (see my comment in here https://github.com/Quitten/Autorize/issues/18) and i will add a default scope.
I am really busy these days so if you can help me write it it would be awesome and I will add you to the contributers list), if not it might take some time.

**Comment by Quitten** [comment]
@Simone1234342 I just pushed https://github.com/Quitten/Autorize/commit/d93fd7490cfbfc5fed9ed9b38ba991b25300abff which allows you do what I described above, closing this issue, let me know if you think different :)
